content/en/docs/concepts/scheduling/scheduling-framework.md

```diff
@@ -140,36 +139,40 @@ to prevent race conditions while the scheduler waits for the bind to succeed.
 
 This is the last step in a scheduling cycle. Once a Pod is in the reserved
 state, it will either trigger [Unreserve](#unreserve) plugins (on failure) or
-[Post-bind](#post-bind) plugins (on success) at the end of the binding cycle.
+[PostBind](#post-bind) plugins (on success) at the end of the binding cycle.
 
-*Note: This concept used to be referred to as "assume".*
+{{< note >}}
+This concept used to be referred to as "assume".
```

Comment:
When did it change / do readers need to know?

Reply:
"assume" is a term used internally. So let me remove this section to avoid confusion.